llvm-spirv/test/extensions/INTEL/SPV_INTEL_runtime_aligned/RuntimeAligned.spt

```diff
@@ -1,3 +1,4 @@
+; XFAIL: *
```

Comment:
Also failing in sycl branch.

Reply:
Do you mean that it already fails in SYCL branch? I'm fine with disabling a test temporarily, but we are currently working on making sure that every `XFAIL`ed test has an associated GitHub issue. Submitting one to investigate this failure later and linking it here in the test would be appreciated

Reply:
> Do you mean that it already fails in SYCL branch?

The CI checks are to be clean, so I don't think this is the case. @MrSidims added llvm-spirv/test/extensions/INTEL/SPV_INTEL_runtime_aligned/RuntimeAligned.spt test to the Khronos repository a week ago (i.e. this is a new test being pulled to the sycl branch). I think what @jsji means is that some changes in the sycl branch breaks the test. Khronos repository CI validation uses vanilla llvm binaries from llvm/llvm-project repository.

> I'm fine with disabling a test temporarily, but we are currently working on making sure that every `XFAIL`ed test has an associated GitHub issue. Submitting one to investigate this failure later and linking it here in the test would be appreciated

That's right. If we can't fix the problem before the merge someone has to investigate and fix the test after the merge. To make this issue visible, it must be reported.

Reply:
Sure. https://github.com/intel/llvm/issues/14427 opened to track.

Reply:
> I think what @jsji means is that some changes in the sycl branch breaks the test. 

Yes, the newly added spt file asserts in sycl branch.